README.md

```diff
@@ -161,8 +154,3 @@ environment:
 to your `docker-compose.yml`.
 
 This generated log file will only hold Invoice Ninja information.
-
-
-### Known issues
-
-Phantomjs doesn't work on linux alpine https://github.com/ariya/phantomjs/issues/14186.
```

Comment:
why did you remove that ?

Reply:
I thought as we use phantomJS in alpine, see line 
```php
RUN cd /usr/share \
    && curl -s -L https://github.com/Overbryd/docker-phantomjs-alpine/releases/download/2.11/phantomjs-alpine-x86_64.tar.bz2 | tar xj \
    && ln -s /usr/share/phantomjs/phantomjs /usr/local/bin/phantomjs
```

it works - or doesn't it?

Reply:
not last time I tried

Reply:
As agreed, the local PhantomJS version is removed from `alpine` version. The cloud version is the preferred one.